src/SprykerEco/Zed/PunchoutCatalogs/Communication/Controller/EntryPointsController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\PunchoutCatalogs\Communication\Controller;
+
+use Generated\Shared\Transfer\PunchoutCatalogEntryPointFilterTransfer;
+use Spryker\Zed\Kernel\Communication\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Persistence\PunchoutCatalogsRepositoryInterface getRepository()
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Business\PunchoutCatalogsFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Communication\PunchoutCatalogsCommunicationFactory getFactory()
+ */
+class EntryPointsController extends AbstractController
+{
+    /**
+     * @uses \SprykerEco\Zed\PunchoutCatalogs\Communication\Controller\IndexController::indexAction()
+     */
+    protected const ROUTE_PUNCHOUT_CATALOGS_CONNECTION_LIST_PAGE = '/punchout-catalogs/';
+
+    protected const PARAM_ID_PUNCHOUT_CATALOG_CONNECTION = 'id-punchout-catalog-connection';
+
+    protected const MESSAGE_CONNECTION_NOT_FOUND = 'Connection not found';
```

Comment:
`ERROR_MESSAGE_CONNECTION_NOT_FOUND` ?

Reply:
Done similarly to all other stories, e.g. https://github.com/spryker-eco/punchout-catalogs/pull/7/files#diff-530ee9e2cbbd2b65737741322a482e9c

Do we have any convention for this?

Reply:
If it is similar to other stories - keep it as it is. We need to keep consistency.